builtin/commit-graph.c

```diff
@@ -248,7 +257,7 @@ static int graph_write(int argc, const char **argv)
 }
 
 int cmd_commit_graph(int argc, const char **argv, const char *prefix)
-{
+{	
```

Comment:
Extra whitespace at end-of-line(s). I use VS Code with the "Trailing Spaces" extension.

Reply:
Also `git log --check` will show whitespace issues and `git rebase --whitespace=fix` will fix most of them.